pytorch_lightning/trainer/trainer.py

```diff
@@ -1956,11 +1956,11 @@ def should_rank_save_checkpoint(self) -> bool:
 
     @property
     def _strategy_type(self) -> _StrategyType:
-        return self._accelerator_connector._strategy_type
+        return self.strategy.distributed_backend
 
     @property
     def _device_type(self) -> _AcceleratorType:
```

Comment:
Should we remove these?

Reply:
Yeah, we should, with other devices related deprecations mentioned #11624. How about we keep it for now to limit the PR size.

Reply:
We can not remove these in a separate PR first. It will break the current accelerator_connector, and it's tough to separate the logic as these properties are intermediate stage which used by the old accelerator_connector selection logic. 
The rest options are we can do it in the current PR or after this PR. It doesn't increase much complexity to keep them for now. Everything after line 721 (the ####) was not used/referred  in the main accel_conn logic, and they were calculated afterward only for backward compatibilities. So how about removing/deprecating after this PR to limit the PR size?